akka-docs/src/main/paradox/persistence.md

```diff
@@ -59,14 +59,14 @@ concepts and architecture of [eventsourced](https://github.com/eligosource/event
  * @scala[`PersistentActor`]@java[`AbstractPersistentActor`]: Is a persistent, stateful actor. It is able to persist events to a journal and can react to
 them in a thread-safe manner. It can be used to implement both *command* as well as *event sourced* actors.
 When a persistent actor is started or restarted, journaled messages are replayed to that actor so that it can
-recover internal state from these messages.
+recover its state from these messages.
  * @scala[`AtLeastOnceDelivery`]@java[`AbstractPersistentActorAtLeastOnceDelivery`]: To send messages with at-least-once delivery semantics to destinations, also in
 case of sender and receiver JVM crashes.
  * `AsyncWriteJournal`: A journal stores the sequence of messages sent to a persistent actor. An application can control which messages
 are journaled and which are received by the persistent actor without being journaled. Journal maintains `highestSequenceNr` that is increased on each message.
 The storage backend of a journal is pluggable. The persistence extension comes with a "leveldb" journal plugin, which writes to the local filesystem.
 Replicated journals are available as [Community plugins](http://akka.io/community/).
- * *Snapshot store*: A snapshot store persists snapshots of a persistent actor's internal state. Snapshots are
+ * *Snapshot store*: A snapshot store persists snapshots of a persistent actor's state. Snapshots are
```

Comment:
I might prefer to keep the 'internal' just to retain actor private / internal state vs being exposed by design. WDYT?

Reply:
My motivation for avoiding "internal" was that I initially thought "internal" meant the Akka framework's internal state of an actor or something, rather than "user-land" internal state.  Maybe it's just me?

Reply:
I think it's better to remove internal for the reasons mentioned.